datalad/interface/common_cfg.py

```diff
@@ -564,6 +564,17 @@
         'default': 'warning',
 
     },
+    'datalad.windows.ssh.client': {
+        'ui': ('question', {
+            'title': "Location of OpenSSH for Windows' ssh.exe",
+            'text': "To use git with ssh on Windows the Windows system ssh"
+                    "client, i.e. OpenSSH for Windows, has to be installed. "
+                    "If it is not installed in the default location, its "
+                    "location can be set in this variable."}),
+        'destination': 'global',
+        'type': EnsureStr(),
+        'default': r'C:\Windows\System32\OpenSSH\ssh.exe',
```

Comment:
why bother making it windows-specific?
Could be just `datalad.ssh.client` defaulting to `"ssh"` on non-windows and this specific path on windows.  If abs path is given, but points to non-existing file, we could even be nice and inform on which paths do have that executable within PATH, something akin to what we have in https://github.com/datalad/datalad/blob/master/tools/which.py ?

note: we do not yet have any other `datalad.windows.` config variable.

Reply:
I introduced a generic "datalad.ssh.executable" configuration. On non-Windows systems it defaults to `ssh`. On Windows systems it defaults to `\Windows\System32\OpenSSH\ssh.exe`, if that path exists, otherwise also to `ssh`.